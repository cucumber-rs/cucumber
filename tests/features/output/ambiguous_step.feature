Feature: ambiguous
  Scenario: ambiguous
    Given foo is 0 ambiguous
