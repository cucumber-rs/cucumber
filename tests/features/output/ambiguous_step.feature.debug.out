ParsingFinished { features: 1, rules: 0, scenarios: 1, steps: 1, parser_errors: 0 }
Started
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Started)
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }, RetryableScenario { event: Started, retries: None }))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }, RetryableScenario { event: Step(Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }, Started), retries: None }))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }, RetryableScenario { event: Step(Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }, Failed(None, None, None, AmbiguousMatch(AmbiguousMatchError { possible_matches: [(HashableRegex(Regex("foo is (\\d+)")), Some(Location { line: 11, column: 1 })), (HashableRegex(Regex("foo is (\\d+) ambiguous")), Some(Location { line: 19, column: 1 }))] }))), retries: None }))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }, RetryableScenario { event: Finished, retries: None }))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3 } }], examples: [], tags: [], position: LineCol { line: 2 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Finished)
Finished
